src/support.cpp

```diff
@@ -1399,12 +1372,24 @@ void AreaSupport::handleTowers(const Settings& settings, Polygons& supportLayer_
     // make tower roofs
     const coord_t layer_thickness = settings.get<coord_t>("layer_height");
     const AngleRadians tower_roof_angle = settings.get<AngleRadians>("support_tower_roof_angle");
-    const double tan_tower_roof_angle = tan(tower_roof_angle);
-    const coord_t tower_roof_expansion_distance = layer_thickness / tan_tower_roof_angle;
     const coord_t tower_diameter = settings.get<coord_t>("support_tower_diameter");
+    const coord_t support_line_width = settings.get<coord_t>("support_line_width");
```

Comment:
type duplication, it is already specified in the `get` template argument
```suggestion
    const auto support_line_width = settings.get<coord_t>("support_line_width");
```